frontend/src/main/scala/bloop/engine/tasks/compilation/CompileGraph.scala

```diff
@@ -185,15 +185,17 @@ object CompileGraph {
         bundle.oracleInputs,
         (_: CompilerOracle.Inputs) => {
           deduplicate = false
-          scheduleCompilation(inputs, bundle, compile)
+          scheduleCompilation(inputs, bundle, client, compile)
         }
       )
 
       if (!deduplicate) {
         ongoingCompilation.traversal
       } else {
         val rawLogger = logger.underlying
-        rawLogger.debug(s"Deduplicating compilation for ${bundle.project.name} (${client}")
+        rawLogger.info(
```

Comment:
I'm not sure a regular user is interested in this log. I think this would be noise for me at least.

Reply:
We can consider disabling it down the road in the next one or two stable releases. For now, I find it important that users know when a deduplication is happening in case there's some kind of misbehavior and want to report it, or in case a compilation is not rightfully deduplicated. I prefer trading log noise over easiness of reporting...